salt/modules/boto3_route53.py

```diff
@@ -718,6 +721,91 @@ def delete_hosted_zone_by_domain(Name, PrivateZone=None, region=None, key=None,
     return delete_hosted_zone(Id=Id, region=region, key=key, keyid=keyid, profile=profile)
 
 
+def aws_encode(x):
+    '''
+    An implementation of the encoding required to suport AWS's domain name
+    rules defined here__:
+
+    .. __: http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/DomainNameFormat.html
+
+    While AWS's documentation specifies individual ASCII characters which need
+    to be encoded, we instead just try to force the string to one of
+    escaped unicode or idna depending on whether there are non-ASCII characters
+    present.
+
+    This means that we support things like ドメイン.テスト as a domain name string
+    per the idna documentation `here <https://pypi.org/project/idna>`_, in
```

Comment:
Sorry, I thought my last review was clear enough to let you know to update the other similar "here" links. This will also need to be changed like in your last commit.